tools/tests/README.md

```diff
@@ -0,0 +1,98 @@
+<!--Copyright (c) Coherent Labs AD. All rights reserved. -->
+# Testing the components
+
+The components are tested using [Jasmine](https://jasmine.github.io/) for assertions
+and checks and [Karma](https://karma-runner.github.io/latest/index.html) for running the tests.
+
+The test.js file located in scripts runs all tests. It builds the components and
+copies the bundle files into the tests folders to ensure that the tests are running
+with the latest version of the components.
+
+To run the test execute:
+
+`npm run test`
+
+in the root of the GameUIComponents.
+
+The test script will execute `karma start tools/tests/karma.conf.js` which will automatically
+run the tests in Google Chrome.
+
+# Adding new tests
+
+All tests are located in tools/tests. There is a separate folder for each component.
+The name of the folder should be the same as the name of the folder that contains the
+source of the component. For example the test folder for the radial-menu should be radial-menu.
+This is required in order to simplify the setup process that copies the source bundles into
+the test folders. All tests are named **test.js**. The `karma.config.js` file
+contains the list of tests that are going tp be executed. If you need to disable a test or
+add a new one - edit the files filed in the configuration.
+
+The actions.js file contains common helper functions that are used in a lot of tests -
+click, nested requestAnimationFrame for the cases where you need to wait a couple of
+frames for the page to be completely rendered before you continue with the test.

```

Comment:
is it "the page to be completely rendered" or maybe the "component" is more accurate statement? Anyway, I'm fine with that, it's generic enough to cover this also, the usage explanation is the most important thing.

Reply:
I meant the page because of the way cohtml parses styles, but in this context it is more appropriate to use components as we're talking about components.